src/data/full-quiz.ts

```diff
@@ -1,4 +1,15 @@
 const fullQuiz = [
+  {
+    Question:
+      "Which one of these data structure follows the FIFO (First In First Out) method?",
+    Answer: "Queue",
+    Distractor1: "Stack",
+    Distractor2: "Array",
+    Distractor3: "Linked List",
+    Explanation:
+      "Queue follows the First In First Out (FIFO) method as the element which gets inserted first gets removed first.",
```

Comment:
The beginning of this sentence would sound better with the letter "A" in front of it. 


```suggestion
      "A Queue follows the First In First Out (FIFO) method as the element which gets inserted first gets removed first.",
```